src/main/java/org/terasology/ligthandshadow/componentsystem/LASUtils.java

```diff
@@ -56,6 +56,9 @@
     public static final Vector3f BLACK_TELEPORT_DESTINATION = new Vector3f(-29, 12, 0);
     public static final String MAGIC_STAFF_URI = "LightAndShadowResources:magicStaff";
 
+    public static final String DEATH_SCREEN = "engine:DeathScreen";
```

Comment:
Why do you have a `gameOverScreen` file if it's never used in code. Holdover from prior attempts?

Reply:
Yup, it is from previous commits. I must have forgotten to remove it.